src/main/resources/templates/admin/cla/link.html

```diff
@@ -147,44 +135,13 @@ <h4 class="modal-title">What does this do?</h4>
 				<button id="submit" type="submit" class="btn btn-success">Use CLA</button>
 			</form>
 		</div>
-		<div layout:fragment="javascript">
 
+		<div layout:fragment="javascript">
 			<script th:src="@{/webjars/select2/dist/js/select2.min.js}" type="text/javascript"></script>
-						<script type="text/javascript">
-								var reloadRepositoryList = function() {
-										$('#repositories').prop('disabled', true);
-										$.getJSON('./link/repositories.json')
-														.done(function (data) {
-																$("#repositories").select2({data: data});
-																sessionStorage.setItem("repositories", JSON.stringify(data));
-																$('#repositories').removeProp('disabled');
-														});
-								}
-
-								if (sessionStorage.getItem("repositories")) {
-										$("#repositories").select2({
-												data: JSON.parse(sessionStorage.getItem("repositories"))
-										});
-								}
-								else {
-										reloadRepositoryList();
-								}
-
-								var openDataUrls = function() {
-										var urls = $(this).data("urls").split(',');
-										$.each(urls, function(i, u) {
-												window.open(u);
-										});
-										return false;
-								};
-
-								$('#reload-repo-list').click(reloadRepositoryList);
-								$('#open-hook-urls').click(openDataUrls);
-								$('#contributing-md-urls').click(openDataUrls);
-								$('#contributing-adoc-urls').click(openDataUrls);
-
-						</script>
-
+			<script th:src="@{/pages/admin/cla/link.js}" type="text/javascript"></script>
+			<script type="text/javascript">
+				adminClaLink.initialize();
```

Comment:
Any chance we can externalize this too? I'd like to externalize everything so we can setup a Content Security Policy http://www.html5rocks.com/en/tutorials/security/content-security-policy/